pyproject.toml

```diff
@@ -60,19 +60,30 @@ metadata.version.provider = "version_provider"
 metadata.version.provider-path = "."
 experimental = true
 
+# build.verbose = true
+# logging.level = "DEBUG"
+
 [tool.scikit-build.sdist]
-# See MANIFEST.in for details
 include = [
-    "VERSION",
-    "LICENSE",
+    "./VERSION",
+    "./_git_commit.txt",
```

Comment:
nit: You will not need the `./` folder prefix here. Also, in Unix-like systems, the misc file is better to be prefixed with `.`, not `_`.

Reply:
> For example, `setuptools-scm` uses `pkg/_version.py`

`.` is not a valid character in Python module names, so they have to use `_`, while the new file is a `txt` file.